Sources/SwiftSyntaxBuilder/generated/BuildableNodes.swift

```diff
@@ -1350,20 +1350,20 @@ extension TupleExprSyntax {
       leadingTrivia: Trivia? = nil, 
       unexpectedBeforeLeftParen: UnexpectedNodesSyntax? = nil, 
       leftParen: TokenSyntax = .leftParenToken(), 
-      unexpectedBetweenLeftParenAndElementList: UnexpectedNodesSyntax? = nil, 
-      unexpectedBetweenElementListAndRightParen: UnexpectedNodesSyntax? = nil, 
+      unexpectedBetweenLeftParenAndElements: UnexpectedNodesSyntax? = nil, 
```

Comment:
While I was making I was thinking, that those `UnexpectedNodesSyntax` in syntax builder don't make that much sense? 
Because we are generating code here and AFAIK `UnexpectedNodesSyntax` is more related to parsing?

Reply:
Since all the other initializers that you use to create nodes, which don’t take a result builder (e.g. `IntegerLiteralExprSyntax` also take `UnexpectedNodesSyntax`, I think it would be inconsistent if you couldn’t specify unexpected nodes for these initializers that take a result builder.

Reply:
I see ✌️